### DISCUSSION TITLE
E2E examples

### DISCUSSION THREAD

**Discussion by RaicuRobert** [GoogleCloudPlatform/cloud-foundation-fabric#2256] [General]
Hello,

Before starting, I want to thank you for this great repository.
It helps a lot to have such resources when starting.

I do have a small gripe though, it is a bit overwhelming trying to set everything up.
Especially at the end when you start having team repositories/projects.

I expect this to become even more pronounced once https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/pull/2254 is merged as it removes some code regarding teams (github workflow, identity impersonation etc).

Are there plans to add a "stage 3 - extended" Readme detailing how to add these?
- a teams folder
  - containing a "Team 0"
    - having a dev env
      - with a github workflow + impersonation
      - code to deploy a "cloud-run-explore" blueprint for example (end goal)
    - having a prod env
      - with a github workflow + impersonation
      - code to deploy a "cloud-run-explore" blueprint for example (end goal)

I would like to contribute but after a week and a bit of trying to understand the modules and code in this repo I am still lost, there is a lot of code to read.
It would be a lot easier to dive in if there was a E2E example where we can just start replacing parts on a "working" project.

It would be even better if it included an example of how to setup the code repo/pipeline to upload a docker image to an artifact repository for the cloud-run setup to use. But it seems a bit out of scope. Though, it would offer the full picture of a possible setup for a startup.

Thank you for your time,
Rob

**Comment by juliocc** [comment]
Ack. @RaicuRobert the team is bit swamped these days. We'll get back to you ASAP.
